CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Change log
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this
+project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). See [MAINTAINERS.md](./MAINTAINERS.md)
+for instructions to keep up to date.
+
+
+## v1.0.0
+
+
+#### Project Rename
+
+* The repo name has changed from `sf-solana` to `firehose-solana`
+* The binary name has changed from `sfsol` to `firesol` (aligned with https://firehose.streamingfast.io/references/naming-conventions)
+
+#### Flags and environment variables
+
+* Renamed `common-blocks-store-url` to `common-merged-blocks-store-url`
+* Renamed `common-oneblock-store-url` to `common-one-block-store-url`
+* Renamed `common-blockstream-addr` to `common-live-blocks-addr`
+* Renamed the `mindreader` application to `reader`
+* Renamed all the `mindreader-node-*` flags to `reader-node-*`
+* Renamed all instances of `deepmind` to `firehose`
```

Comment:
Let's add specific flags change, document under it on a second level of indentation (please validate the rendering of the Markdown to be sure you got it right:

```
- Level 1
    - Level 2.1
    - Level 2.2
```

Add
- `reader-node-firehose-batch-files-path` => `reader-node-firehose-batch-files-path`